### DISCUSSION TITLE
Camel bottom door prevents bots from entering

### DISCUSSION THREAD

**Discussion by Arctomachine** [FakeFishGames/Barotrauma#14186] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

New implementation of bottom door logic now prevents bot from entering. They keep trying to open inner door, which is not opening manually, only by button.

### Reproduction steps

1. Get bot outside
2. Give command or just wait for bot to return
3. Bot enters airlock and tries to open inner door manually

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.5.8.0 (Summer Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by pavel4win** [comment]
This is a bug in the new camel.
As long as the bottom hatch is open, the airlock door cannot be opened  by the door's button from the outside or the inside.
This door can only be opened by pressing a separate button inside the airlock or button outside the submarine.
If someone exits the camel, does not close the bottom hatch and dies outside, no one will be able to exit the camel until they change the airlock door wiring.
Also, several components are easily accessible and can be easily detached by any ruiner.
Let's hide it all in a circuit box.

**Comment by Regalis11** [comment]
This has been fixed in our private repo now, fix coming up in a future unstable build. Sorry for the inconvenience!
